.buildconfig.yml

```diff
@@ -380,3 +380,7 @@ projects:
     path: samples/dataprotect
     description: 'An app demoing how to use the Dataprotect component to load and store encrypted data in SharedPreferences.'
     publish: false
+  tooling-glean-gradle:
```

Comment:
This was `tooling-glean-gradle-plugin`, but that exceeds the 25-character limit for taskcluster tasks.  This is now a little inconsistent with the name elsewhere.  Should I also remove `-plugin` from the directory names etc.?

Reply:
I have no strong opinion on this, @pocmo might :)